graph/db/sql_store.go

```diff
@@ -1666,12 +1668,12 @@ func (s *SQLStore) IsZombieEdge(chanID uint64) (bool, [33]byte, [33]byte) {
 		return nil
 	}, sqldb.NoOpReset)
 	if err != nil {
-		// TODO(elle): update the IsZombieEdge method to return an
-		// error.
-		return false, route.Vertex{}, route.Vertex{}
+		return false, route.Vertex{}, route.Vertex{},
```

Comment:
nit: we could use the same error for kv and sql.

Reply:
cool - done